modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -97,4 +97,1235 @@ Status description:: error: connection exception - alias chains are not permitte
 
 === 08N15
 
-Status description:: error: connection exception - no such routing policy. Policy definition of the routing policy { $routingPolicy } could not be found. Verify that the spelling is correct.
\ No newline at end of file
+Status description:: error: connection exception - no such routing policy. Policy definition of the routing policy { $routingPolicy } could not be found. Verify that the spelling is correct.
+
+[[_data-exceptions]]
+== Data exceptions
+
+The following is a list of the data exceptions that can occur in Neo4j.
```

Comment:
@Lojjs, could you provide an intro sentence for data exceptions?

Reply:
I can try :) something like this maybe:

> Database exceptions occur when a client request contains wrong format or types, or other unsupported input. Some examples are data and constraint creation which conflicts with already existing constraints, properties of non-storable type, and spatial and temporal values with invalid components. 

This class seems quite broad, but I thought mentioning a few examples might give the reader an idea about which sort of errors are covered.